pypeit/wavecalib.py

```diff
@@ -279,8 +285,10 @@ def print_diagnostics(self):
         # RMS
         diag['RMS'] = [0 if wvfit.rms is None else wvfit.rms for wvfit in self.wv_fits]
         diag['RMS'].format = '0.3f'
```

Comment:
How about adding a couple more diagnostics?

- The minimum and maximum pixel of lines ID'd and fitted (gives a sense of how much extrapolation)
- The number of lines ID'd

Reply:
And I took the initiative to add `fwhm` which we've been thinking about including for some time.

Reply:
Actually, I am making that a side PR into this branch

Reply:
added min max wavelength of lines ID'd and fitted. Also added the wavelength coverage (%) of those line.